mllib/src/test/java/org/apache/spark/ml/param/JavaTestParams.java

```diff
@@ -108,9 +108,6 @@ private void init() {
     myIntParam_ = new IntParam(this, "myIntParam", "this is an int param", ParamValidators.gt(0));
     myDoubleParam_ = new DoubleParam(this, "myDoubleParam", "this is a double param",
       ParamValidators.inRange(0.0, 1.0));
-    List<String> validStrings = Arrays.asList("a", "b");
```

Comment:
why remove?

Reply:
When we have `T: ClassTag`, then java code cannot define the class like `Param[XXX]`, java compiler cannot generate class tag.